pkg/app/piped/cloudprovider/kubernetes/diff.go

```diff
@@ -68,7 +71,17 @@ func DiffList(olds, news []Manifest, opts ...diff.Option) (*DiffListResult, erro
 	return cr, nil
 }
 
-func (r *DiffListResult) DiffString() string {
+type DiffRenderOptions struct {
+	MaskSecret    bool
+	MaskConfigMap bool
+	// Maximum number of changed manifests should be shown.
+	// Zero means rendering all.
+	MaxChangedManifests int
```

Comment:
how about `PrintableDiffTotal` or `MaxPrintableDiff` 👀

Reply:
I get your point. But this field is inside the `DiffRenderOptions` struct which is only for rendering, so do we need to emphasize that again?

Reply:
Ah okay, I get your point 👍 then lets keep it as is 🙆‍♀️